lib/FusionInventory/Agent/Task/Inventory/Win32/Drives.pm

```diff
@@ -87,9 +89,48 @@ sub _getDrives {
             TYPE        => $type[$object->{DriveType}],
             VOLUMN      => $object->{VolumeName},
         };
+
+        $seen{$object->{DeviceID} || $object->{Caption}} = 1;
+    }
+
+    # Scan Win32_Volume to check for mounted point drives
+    foreach my $object (getWMIObjects(
+        class      => 'Win32_Volume',
+        properties => [ qw/
+            InstallDate Description FreeSpace FileSystem Name Caption DriveLetter
+            SerialNumber Capacity DriveType Label
+        / ]
+    )) {
+        # Skip volume still seen as Win32_LogicalDisk class
+        if (@drives && exists($object->{DriveLetter}) && $object->{DriveLetter}) {
+            $seen{$object->{DriveLetter}}
```

Comment:
Hi David, yes they can, but you can mount a drive over a folder like over C:\PathToMountPoint\ and this was not reported.